docs/forwarding_logs_to_gateway.md

```diff
@@ -0,0 +1,119 @@
+# Forwarding Logs to Loki Operator
+
+This document will describe how to send application, infrastructure, and audit logs to the Lokistack Gateway as different tenants using Promtail. The gateway provides secure access to the distributor (and query-frontend) via consulting an OAuth/OIDC endpoint for the request subject.
+
+__Please read the [hacking_loki_operator.md](./hacking_loki_operator.md) document before proceeding with the following instructions.__
+
+_Note: While this document will only give instructions for two methods of log forwarding into the gateway, the examples given in the Promtail section can be extrapolated to other log forwarders._
+
+## Openshift Logging
+
+Although there is a way to [forward logs to an external Loki instance](https://docs.openshift.com/container-platform/4.9/logging/cluster-logging-external.html#cluster-logging-collector-log-forward-loki_cluster-logging-external), [Openshift Logging](https://github.com/openshift/cluster-logging-operator) does not currently have support to send logs through the Lokistack Gateway.
+
+Support will be added in the near future.
+
+## Forwarding Clients
+
+In order to enable communication between the client(s) and the gateway, follow these steps:
+
+1. Deploy the Loki Operator and an `lokistack` instance with the gateway enabled.
```

Comment:
Need to precise what `with the gateway enabled` means, i.e. your operator should run with the appropriate feature flags.